ccm_web/server/src/invitation/cirrus-invitation.service.ts

```diff
@@ -26,8 +26,12 @@ export class CirrusInvitationService {
     private readonly configService: ConfigService<Config, true>,
     private readonly httpService: HttpService
   ) {
+    const apiURL = 'https://apps.cirrusidentity.com/console'
+    const cirrusAPIVersion = '/api/v1/'
+    const CirrusAPIEndPoint = '/batchInviteCsv'
```

Comment:
I'd probably move these constants to `cirrus.interfaces`? Or maybe just out of the class definition.

Reply:
OK.  I could see that.

Reply:
🏁